samples/sample_hevc_fei/include/hevc_fei_preenc.h

```diff
@@ -92,6 +96,8 @@ class FEI_Preenc : public IPreENC
 
     mfxStatus PreEncFrame(HevcTask * task);
 
+    mfxStatus ResetIOState();
```

Comment:
Add virtual because it is virtual. Same for 141.

Reply:
See no reason to make it virtual because other functions in this class are non-virtual and there's no need to derive from it now.

Reply:
This method overrides parent virtual IPreENC::ResetIOState. It is already virtual.